CMakeLists.txt

```diff
@@ -21,12 +21,23 @@ else()
   set(CMAKE_BUILD_TYPE "RelWithDebInfo")
 endif()
 
+if (DEFINED ENV{QUICER_USE_OPENSSL3})
+  message(STATUS "Use openssl3")
+  set(QUIC_TLS "openssl3")
+endif()
+
 if (DEFINED ENV{QUIC_ENABLE_LOGGING})
   set(QUIC_ENABLE_LOGGING $ENV{QUIC_ENABLE_LOGGING})
+  set(QUIC_LOGGING_TYPE lttng)
 else()
   set(QUIC_ENABLE_LOGGING "OFF")
 endif()
 
+if (DEFINED ENV{QUIC_LOGGING_TYPE})
+  set(QUIC_ENABLE_LOGGING "ON")
```

Comment:
`QUIC_LOGGING_TYPE` can activate `QUIC_ENABLE_LOGGING` even though this one is not set. And vice versa. Therefore, shouldn't be better to merge this two variables in a single one? Like `QUIC_LOGGING` which could be `off`, `lttng`, etc (any value different from `off` would be considered the logging type and would imply that logging is on)?

Reply:
good idea but these are exposed by msquic and they are optional config,  I think it is just simple to keep them aligned with MsQuic.